identity/src/main/java/com/android/identity/internal/CborUtil.java

```diff
@@ -1,10 +1,11 @@
-package com.android.identity;
+package com.android.identity.internal;
```

Comment:
So `Util.java` is a hot mess but I think `CborUtil` is actually fine and could be moved to the `util` package so apps can use it too.

Reply:
done